File: CoreJava/iemotionable-app/src/com/techlab/human/Boy.java
package com.techlab.human;

import com.techlab.interfaces.IEmotionable;

public class Boy implements IEmotionable {

	@Override
	public void cry() {
		System.out.println("Boy is Crying...");

	}

	@Override
	public void laugh() {
		System.out.println("Boy is Laughing...");
	}

}

File: C#Basics/Interface-App/Human-Behaviour-App/Boy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Human_Behaviour_App
{
    class Boy : IEmotionable
    {
		void IEmotionable.cry()
		{
			Console.WriteLine("Boy is Crying...");

		}

		
		 void IEmotionable.laugh()
		{
			Console.WriteLine("Boy is Laughing...");
		}
	}
}
